src/components/item-filter/index.tsx

```diff
@@ -0,0 +1,214 @@
+import { useState, useEffect } from 'react';
+import clsx from 'clsx';
+import NiceSelect from '@ui/nice-select';
+import { Range } from 'react-range';
+import { IRenderTrackParams } from 'react-range/lib/types';
+import { selectFilteredItems } from '../../features/leda-nft/store/leda-nft.slice';
+import useAppSelector from '../../store/hooks/useAppSelector';
+import SliderTrack from '../ui/input-range/slider-track';
+import SliderThumb from '../ui/input-range/slider-thumb';
+
+type Props = {
+  setNfts: any;
+};
+
+type PriceRangeType = {
```

Comment:
Can you put these types in a different file? It could be in a single file with all these types.